.github/workflows/test.yaml

```diff
@@ -39,7 +39,10 @@ jobs:
 
     steps:
       - name: "Checkout Code"
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      - name: "setup mysql"
+        run: sudo /etc/init.d/mysql start mysql -h 127.0.0.1 --port 33306 -e 'CREATE DATABASE sqlxtest;' -uroot -pmysql
```

Comment:
> MYSQL_DATABASE
This variable is optional and allows you to specify the name of a database to be created on image startup. If a user/password was supplied (see below) then that user will be granted superuser access ([corresponding to GRANT ALL](http://dev.mysql.com/doc/en/adding-users.html)) to this database.

I think we can skip this part.